swagger/api/v1/presentation_statuses.yaml

```diff
@@ -0,0 +1,56 @@
+get:
+  operationId: getPresentationStatus
+  summary: Gets the status of the presentation for the presentation ID.
+  parameters:
+    - name: id
+      in: path
+      description: Presentation ID
+      required: true
+      schema:
+        type: string
+  responses:
+    '200':
+      description: Status has been provided.
+      content:
+        application/json:
+          schema:
+            type: object
+            $ref: './api/models/status.yaml#/presentation_status'
+    '400':
```

Comment:
Given this is a REST API with an id in the path, I would also expect an explicit 404 response

Reply:
added